### PR TITLE
[crmsh-4.1] Change lock directory under /run

### PR DESCRIPTION
**Pull request by liangxin1300**
backport #772

### CODE REVIEW

File: crmsh/lock.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ class Lock(object):
 
     LOCK_DIR = "/tmp/.crmsh_lock_directory"
     MKDIR_CMD = "mkdir {}".format(LOCK_DIR)
-    RM_CMD = "rm -rf {}".format(LOCK_DIR)
+    RM_CMD = "rm -rf {};sync".format(LOCK_DIR)
```
Comment (zzhou1) [inline crmsh/lock.py:33]: Two comments,
1. sync is the system wide, likely overkill the whole system. I would avoid using it.
2. /run is gurenteed to be cleaned per boot, and fits your finding well. Seems, /run/crm is created by some one. If it is created by crmsh.rpm, then, it's safe to use, 
LOCK_DIR = "/run/crm/.crmsh_lock_directory"

Comment (liangxin1300) [reply crmsh/lock.py:33]: Good suggestions! Already tried to use `/run/crm/.crmsh_lock_directory`, seems work

**Comment by liangxin1300** [comment]
@arbulu89 What do you think?

File: crmsh/lock.py (hunk 1 above)
Comment (liangxin1300) [reply crmsh/lock.py:33]: @zzhou1 `/run/crm` is created just when pacemaker service starting, not related with crmsh

Comment (zzhou1) [reply crmsh/lock.py:33]: > @zzhou1 `/run/crm` is created just when pacemaker service starting, not related with crmsh

Then, it is is safe for crmsh to use it directly.

**Comment by liangxin1300** [comment]
> Refer to the existing directory of /run, it does not need hide the inode LOCK_DIR.
> I prefer something like /run/crmsh_bootstrap_lock

@zzhou1  By design, lock.py was a common used file lock mechanism, so I think it's suitable to keep the original file name

**Review by arbulu89 — APPROVED** [review]

**Pull request merged**